packages/ARU/PKGBUILD

```diff
@@ -0,0 +1,21 @@
+pkgname=kapitan
+pkgver=0.27.0
```

Comment:
Can you pin this to the one in [version.py](https://github.com/deepmind/kapitan/blob/master/kapitan/version.py#L11)?